packages/stores/src/mockStore.ts

```diff
@@ -262,6 +266,7 @@ const mock = (): TStores & { is_mock: boolean } => {
             changeCurrentLanguage: jest.fn(),
             changeSelectedLanguage: jest.fn(),
             is_network_online: false,
+            services_error,
```

Comment:
In `common-store.js` `services_error` is `{}` by default. Shouldn't you add it here?

Reply:
defined on line 17, as `const services_error = { code: '', message: '', type: '' };`

i got this from one of your PRs